Gems/Atom/Feature/Common/Code/Source/Mesh/MeshFeatureProcessor.cpp

```diff
@@ -1363,6 +1716,73 @@ namespace AZ
             m_flags.m_needsInit = false;
         }
 
+        struct MeshInstancingSupport
+        {
+            bool m_canSupportInstancing = false;
+            bool m_isTransparent = false;
+        };
+
+        static MeshInstancingSupport CanSupportInstancing(
+            Data::Instance<RPI::Material> material, bool useForwardPassIbleSpecular, const RHI::DrawListTag& transparentDrawListTag)
+        {
+            MeshInstancingSupport result;
+            if (useForwardPassIbleSpecular)
+            {
+                // Forward pass ibl specular uses the ObjectSrg to set the closest reflection probe data
+                // Since all instances from a single instanced draw call share a single ObjectSrg, this
+                // will not work with instancing unless they happen to all share the same closes probe.
```

Comment:
I think this will disable instancing for low end devices that want IBL.  Is the implication here we can pass reflection probe data as part of instancing buffer?